core/src/main/java/tc/oc/pgm/api/party/Party.java

```diff
@@ -14,16 +15,14 @@
 import tc.oc.pgm.filters.query.Query;
 import tc.oc.pgm.match.ObservingParty;
 import tc.oc.pgm.util.chat.Audience;
-import tc.oc.pgm.util.component.Component;
-import tc.oc.pgm.util.named.Named;
 
 /**
  * A group of related {@link MatchPlayer}s in a {@link Match}.
  *
  * @see Competitor for participating {@link MatchPlayer}s.
  * @see ObservingParty for observing {@link MatchPlayer}s.
  */
-public interface Party extends Audience, Named {
+public interface Party extends Audience {
```

Comment:
It makes sense for `Party` to extend `Named`.

Maybe `Named` needs a simple `getName()` method.